pymatgen/transformations/advanced_transformations.py

```diff
@@ -1874,31 +1874,33 @@ def _proj(b, a):
 
 
 class SQSTransformation(AbstractTransformation):
-    """A transformation that creates a special quasirandom structure (SQS) from a structure with partial occupancies."""
+    """A transformation that creates a special quasi-random structure (SQS)
+    from a structure with partial occupancies.
+    """
 
     def __init__(
         self,
-        scaling,
-        cluster_size_and_shell=None,
-        search_time=60,
-        directory=None,
-        instances=None,
-        temperature=1,
-        wr=1,
-        wn=1,
-        wd=0.5,
-        tol=1e-3,
-        best_only=True,
-        remove_duplicate_structures=True,
-        reduction_algo="LLL",
+        scaling: int | list[int],
+        cluster_size_and_shell: dict[int, int] | None = None,
+        search_time: float = 60,
+        directory: str | None = None,
+        instances: int | None = None,
+        temperature: float = 1,
+        wr: float = 1,
+        wn: float = 1,
+        wd: float = 0.5,
+        tol: float = 1e-3,
+        best_only: bool = True,
+        remove_duplicate_structures: bool = True,
+        reduction_algo: Literal["niggle", "LLL"] = "LLL",
     ):
         """
         Args:
             scaling (int or list): Scaling factor to determine supercell. Two options are possible:
-                    a. (preferred) Scales number of atoms, e.g., for a structure with 8 atoms,
-                       scaling=4 would lead to a 32 atom supercell
-                    b. A sequence of three scaling factors, e.g., [2, 1, 1], which
-                       specifies that the supercell should have dimensions 2a x b x c
+                a. (preferred) Scales number of atoms, e.g., for a structure with 8 atoms,
+                    scaling=4 would lead to a 32 atom supercell
+                b. A sequence of three scaling factors, e.g., [2, 1, 1], which
+                    specifies that the supercell should have dimensions 2a x b x c
             cluster_size_and_shell (Optional[Dict[int, int]]): Dictionary of cluster interactions with entries in
                 the form number of atoms: nearest neighbor shell
         Keyword Args:
```

Comment:
@esoteric-ephemera Could you check if this doc string is incorrect?

> Currently supported options are "niggli" or "LLL". "False" does not reduce structure.

Does `"False"` actually changes the behavior? Don't have ATAT installed myself.

Reply:
The structure reduction is actually done through PMG's `IStructure.get_reduced_structure` function, which only supports "niggli" or "LLL". Passing "False" would return a ValueError, but passing `False` (bool) would skip structure reduction. That docstr should probably be amended

Reply:
will do